nio.conf

```diff
@@ -230,9 +230,16 @@ secure=[[PK_SECURE]]
 #
 #websocket_ping_interval=10
 
-# List of all the brews you wish to use with the system
-# should be a comma separated list of the classes
-#
+# List of all the Pubkeeper brews you wish to use in this instance
+# This should be a comma separated list of unique brew identifiers. If you only are
+# using one instance of a particular brew class, you may list the full module path of
+# that brew class. Note, the brew module must begin with "pubkeeper.brew" for this to work
+# You can configure the options for the brew in a separate configuration section with a 
+# title of "pubkeeper.brew.brew_identifier" where "brew_identifier" is the item in this list.
+#
+# Example: brews=pubkeeper.brew.local.brew.LocalBrew, websocket1, websocket2
+#   This example will create 3 brews in this instance. One local brew and two websocket
+#   brews whose details will be configured in the [pubkeeper.brew.websocket1] config section.
```

Comment:
I don't really like the mix-match of schemes here.  For the first would be the directive-name fully-qualified, and the other are prefixed?

I'd probably advocate for the list here to be a list of directives, which by default specifies the `brew_class` which you can override the `brew_class` element of that directive.

Reply:
Hmm ok I can probably do that. The thinking was you would want to avoid namespace collisions with other configuration sections but it does reduce complexity by just having one naming scheme. I'll give it a shot